submission/notifications/README.md

```diff
@@ -0,0 +1,2 @@
+# Notification system
+1. Notification micro-module contains logic that helps to dispatch notification to users.
```

Comment:
which contains

Reply:
do you mean? `Notification micro-module which contains` ?